packages/@okta/vuepress-site/docs/guides/registration-inline-hook/add-registration-code/nodejs/get-submitted-credentials.md

```diff
@@ -0,0 +1,14 @@
+The following code represents a simple check on the email property of an incoming request from Okta; that is, the email used to self-register. The values of `data.userProfile` from within the request body contains the email property that will be verified. The variable `emailRegistration` stores the value of the email.
```

Comment:
```suggestion
The following code represents a simple check on the `email` property of an incoming request from Okta; that is, the email used to self-register. The values of `data.userProfile` from within the request body contains the `email` property that will be verified. The variable `emailRegistration` stores the value of the email.
```

Since I assume we're talking about the actual property in the object